echo/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
+spring.application.name=echo-service
 
+server.port=8081
+
+eureka.client.serviceUrl.defaultZone=${EUREKA_URI:http://localhost:8761/eureka}
+eureka.client.instance.preferIpAddress=true
```

Comment:
This lets Eureka to advertise the IP addresses of services rather than the hostname [ref](https://cloud.spring.io/spring-cloud-netflix/multi/multi_spring-cloud-eureka-server.html#spring-cloud-eureka-server-prefer-ip-address). It depends on cases.